gallery/src/styles/NxPageTitle/NxPageTitleActionsExample.html

```diff
@@ -0,0 +1,28 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+<div class="nx-page-title nx-page-title__actions">
+  <h1 class="nx-h1">
+    <svg class="nx-icon nx-page-title__page-icon" style="height: 1em; width: 1em;" viewBox="-1 -1 2 2">
+      <circle r="1" />
+    </svg>
+    Page Title
+  </h1>
+  <div class="nx-btn-bar">
+    <button class="nx-btn">First</button>
```

Comment:
I think we need design input on what types of buttons should be used here.  I think in most mockups I've seen it's the tertiary buttons.  The blue secondary ones can look a little out of place. @dturner32

Reply:
> I think we need design input on what types of buttons should be used here. I think in most mockups I've seen it's the tertiary buttons. The blue secondary ones can look a little out of place. @dturner32

I changed them to tertiary, but IMO that's going to be up to the designers: https://github.com/sonatype/sonatype-react-shared-components/pull/231/commits/875b8850526b34ec9f9d9e4dd017d39532dbf51c

Reply:
> I think we need design input on what types of buttons should be used here. I think in most mockups I've seen it's the tertiary buttons. The blue secondary ones can look a little out of place. @dturner32

I'd advise using tertiary buttons in this case